newrelic-daemon/Dockerfile

```
@@ -0,0 +1,17 @@
+# alpine:latest at 2019-01-30T19:08:47IST
+FROM alpine@sha256:46e71df1e5191ab8b8034c5189e325258ec44ea739bba1e5645cff83c9048ff1
+
```

Need to add EasyEngine labels for proper cleanup and `ee cli self-uninstall` to work.
```suggestion

LABEL maintainer="Mriyam Tamuli <mbtamuli@gmail.com>"
LABEL org.label-schema.schema-version="1.0.0-rc1"
LABEL org.label-schema.vendor="EasyEngine"
LABEL org.label-schema.name="newrelic-daemon"

```